src/main/resources/swagger/api-docs.yaml

```diff
@@ -533,6 +547,32 @@ paths:
       summary: Query to see if you're a library curator
       operationId: isCurator
 
+  /api/library/{workspaceNamespace}/{workspaceName}/discoverableBy:
```

Comment:
nitpick on endpoint name - I think it should be more noun-like for convention. `discoverableGroups` seems most appropriate since you're using that term already.